website/docs/r/virtual_machine.html.markdown

```diff
@@ -144,6 +161,26 @@ The following attribute are exported:
 * `state` - State of the virtual machine.
 * `lcmstate` - LCM State of the virtual machine.
 
+## Instantiate from a template
```

Comment:
This is quite confusing for this first version.

If the user instantiate a `template_id` here is the behavior:
* for all parameters excepted `context`: parameters present in VM overrides parameters defined in template
* for context: it merges them

Thanks